content/https/https-3/contents.lr

```diff
@@ -0,0 +1,14 @@
+_model: question
+---
+title: What's the difference between Tor and DoH (DNS over HTTPS)?
+---
+seo_slug: difference-between-Tor-DoH
+---
+description:
+DNS over HTTPS (DoH) is a protocol for performing remote DNS resolution via the HTTPS protocol.
+A goal of the method is to increase user privacy and security by preventing eavesdropping and manipulation of DNS data by man-in-the-middle attacks by using the HTTPS protocol to encrypt the data between the DoH client and the DoH-base DNS resolver. 
+Encryption by itself does not protect privacy, encryption is simply a method to obfuscate the data.
+DoH specifies how to send and receive DNS queries over HTTPS. Server configuration is performed out of band, and the connection with the resolver is secured as any other HTTPS traffic. DoH is mostly targeted at web browsers and does not have the potential for improving the privacy properties of transactions between recursive resolvers and authoritative nameservers.

```

Comment:
While it's true that DoH "does not have the potential for improving the privacy properties of transactions between recursive resolvers and authoritative nameservers," (i haven't seen anyone propose using it for the "second hump" of the DNS camel) it also doesn't preclude any such improvement.  So this sentence is sort of like saying "DoH does not have the potential for improving the privacy properties of video conferencing".